Rename Valid language tag to Known primary language tag (#1738)
act-rules/act-rules.github.io: _rules/html-page-lang-xml-lang-match-5b7ae0.md

#### Inapplicable Example 9

> The `lang` attribute of this page is an [iso 639.2][] three letters code, hence not a [known primary language tag][].

Reviewer: ```suggestion
The `lang` attribute of this page is an [iso 639.2][] three letters code, hence has no [known primary language tag][].
```

```html
<html lang="eng" xml:lang="en"></html>
```

#### Inapplicable Example 10

> The `lang` attribute of this page is a [grandfathered tag][grandfathered tags], hence not a [known primary language tag][].

Reviewer: ```suggestion
The `lang` attribute of this page is a [grandfathered tag][grandfathered tags], hence has no [known primary language tag][].
```
